.github/workflows/build_release.yml

```diff
@@ -15,7 +15,19 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-20.04, macos-14, windows-latest]
+        include:
+          - os: ubuntu-20.04
+            arch: x86_64
+            name: x86-ubuntu-20.04
+          - os: ubuntu-24.04-arm64
+            arch: aarch64
+            name: aarch64-ubuntu-24.04
```

Comment:
If we are going to go with friendly arch names names here in the `name` field I think `arm64` is more readable then `aarch64`.